CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [9.0.0] (IN PROGRESS)
 
+## [8.1.0](https://github.com/folio-org/ui-tags/tree/b8.1)
+[Full Changelog](https://github.com/folio-org/ui-tags/compare/v8.0.2...b8.1)
```

Comment:
```suggestion
[Full Changelog](https://github.com/folio-org/ui-tags/compare/v8.0.2...v8.1.0)
```

Reply:
Please include the release date. Up to you whether to correct the v8.0.2 release date as well (it was 2023, not 2022).